### ISSUE TITLE
Node-webkit for "native" application client?

### ISSUE THREAD

**Issue by Fire-Dragon-DoL** [benweet/stackedit#615]
This may be a good idea, using [Node-webkit](https://github.com/rogerwang/node-webkit) to provide a _native_ client. It supports an API to write files locally, which may be interesting if it's run locally. It also has an `app://` protocol, allowing you to serve everything from root path (so there are no issues with relative paths), it's available for linux, windows and OSX

**Comment by basicserge** [comment]
+1

**Comment by AlJohri** [comment]
+1 this would be amazing

**Comment by Rickgg** [comment]
Hello all! I'm currently developing [StackedIt Desktop Edition.](https://github.com/Rickgg/stackedit-desktop) The instructions for running are in [Developer's Guide.](https://github.com/rickgg/stackedit-desktop/blob/master/doc/developer-guide.md#developer-guide)

**Comment by AlJohri** [comment]
@Rickgg link 404's

**Comment by Rickgg** [comment]
Yes, sorry about that. It was throwing me a weird error that wouldnt let it start. I'm working on a new markdown editor. If you want, I can make it again, it isn't so hard.
